src/ooo_cpu.cc

```diff
@@ -813,7 +814,7 @@ void LSQ_ENTRY::finish(std::deque<ooo_model_instr>::iterator begin, std::deque<o
 
 void LSQ_ENTRY::finish(ooo_model_instr& rob_entry) const
 {
-  assert(rob_entry.instr_id == this->instr_id);
+  // assert(rob_entry.instr_id == this->instr_id);
```

Comment:
```suggestion
  assert(rob_entry.instr_id == this->instr_id);
```

Is this left over from some earlier change? We probably should leave it uncommented if possible.

Reply:
Oops my bad! I forgot to uncommented it out